<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeWidget</class>
 <widget class="QWidget" name="UpgradeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>相机升级</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>500</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_ServerIp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>192.168.0.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitleServerIP">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>70</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>服务器IP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>升级</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Del">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清空任务文件</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_CLeanBinFiles">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>130</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清空bin目录</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>服务器</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonUDisk">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>U盘</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;提示：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt;&quot;&gt;1.升级过程中请不要断开电源&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;2.选中清空任务文件，将删除相机里全部的任务文件&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;3.选中清空bin目录，将删除bin目录下的全部文件&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;（注：bin目录存放的是xml文件、库文件和可执行文件等）&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; font-weight:600; background-color:#ffffff;&quot;&gt;升级流程如下：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;1.服务器PC上开启tftp服务&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;2.把升级包update.tar.gz放入tftp指定的服务目录下&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;3.根据需要选择是否清空任务文件或bin目录&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;4.输入服务器IP，点击升级&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;（请确保相机和服务器PC处于同一网段内）&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:20.02px; background-color:#ffffff;&quot;&gt;&lt;span style=&quot; font-family:'幼圆'; font-size:12pt; background-color:#ffffff;&quot;&gt;5.等待进度条提示掉电重启相机后，掉电重启相机即可&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
